src/Parser/Shaman/Restoration/Modules/Spells/HealingWave.js

```diff
@@ -0,0 +1,37 @@
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import Combatants from 'Parser/Core/Modules/Combatants';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+
+class HealingWave extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    combatants: Combatants,
+  };
+
+  get suggestedThreshold(){
```

Comment:
Please consider also adding a suggestion for this so the suggestions have at least the same info as the checklist but in a different view.